handbook/digital-experience/README.md

```diff
@@ -485,6 +485,15 @@ It's not enough to just "delete" a recording of a meeting in Gong.  Instead, use
 - Always check back to ensure the recording **and** transcript were both deleted.
 
 
+### Failed Gong recording
+In some instances, Gong may not record a call and it will go unnoticed throughout the meeting. A local recording will be made and then it will need to be shared to the related parties.
+
+- The Apprentice will schedule a meeting with the CEO.
+- They will then take the local recording and share it in slack.
```

Comment:
```suggestion
- Share the local recording in slack.
```
- Who is "they"? Remember the audience is you.
- Share where in slack and to who? Do I at mention someone? how do I share it exactly?